src/Stats/index.jsx

```diff
@@ -1,33 +1,103 @@
-import React from 'react';
-import { View, ScrollView } from 'react-native';
-import { ListItem } from 'react-native-elements';
+import React, { useState } from 'react';
+import { View, ScrollView, Text } from 'react-native';
+import { ListItem, Overlay, Button } from 'react-native-elements';
 import { useSelector } from 'react-redux';
-import moment from 'moment';
-import utils from './utils';
 import styles from '../Common/themeStyle';
+import MainHeader from '../Common/MainHeader';
+import LargeButton from '../Common/LargeBtn';
+import GradientIcon from '../Common/icon';
+import utils from './utils';
+import SubHeader from '../Common/SubHeader';
+
+const moment = require('moment');
 
 export default function Stats() {
   const { transactions } = useSelector((state) => state.transactions);
+  const [currentMonthSet, setCurrentMonthSet] = useState(utils.getDateSet(moment(), 'months'));
+  const [currentYearSet, setCurrentYearSet] = useState(utils.getDateSet(moment(), 'years'));
+  const [view, setCurrentView] = useState('month');
+  const [overlay, setOverlay] = useState(false);
+
+
+  const ifTransition = (arr) => (arr.length !== 0);
```

Comment:
what kind of transition is this? Do you really need this function at all?

Reply:
Remove this function and judging if transition exists in the component

Reply:
actually this method just returns whether the given array is empty or not, there is no transition at all, it could be simply isEmpty, or i might don't understand whats going on here. 
Also if you use this method only once then you dont need to define it at all, you can use the condition instead of a function call.

Reply:
Have put it in the return component as `(transactions.length !== 0)`